### PR TITLE
Update Qplane fltmodes with dedicated pages

### PR DESCRIPTION
**Pull request by Hwurzburg**
in order to add QACRO to the toctree and eliminate the build error, ALL flight modes have to be added....but each needs its own page, not a reference to Copter....and there ARE numerous differences....
After a good deal of work, the pages have been proofed, built, and XREFs checked....but being rather dyslexic, that's no guarantee...

But I think this greatly improves the quadcopter wiki...at least a lot more params get explained....

### CODE REVIEW

File: plane/source/docs/qhover-mode.rst [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+In QHOVER mode, QuadPlane maintains a consistent altitude while
+allowing roll, pitch, and yaw to be controlled normally. This page
+contains important information about using and tuning alt hold.
+
+Overview
+========
+
+When altitude hold mode (aka QHOVER) is selected, the throttle is
+automatically controlled to maintain the current altitude. Roll, Pitch
```
Comment (tridge) [inline plane/source/docs/qhover-mode.rst]: I think it should say that throttle controls climb rate, with mid-throttle meaning hold height

File: plane/source/docs/qhover-mode.rst [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+(:ref:`QLOITER <qloiter-mode>`, etc.) so the
+information here pertains to those modes as well.
+
+.. note::
+
+   The flight controller uses a barometer which measures air pressure
+   as the primary means for determining altitude ("Pressure Altitude") and
+   if the air pressure is changing in your flight area due to extreme
+   weather, the QuadPlane will follow the air pressure change rather than
+   actual altitude.  When fitted and enabled, a downward facing rangefinder
```
Comment (tridge) [inline plane/source/docs/qhover-mode.rst]: rangefinder can't be used for height control in hover yet

File: plane/source/docs/qland-mode.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
+.. _qland-mode:
+
+==========
+QLAND Mode
+==========
+
+QLAND Mode attempts to bring the QuadPlane straight down descending to :ref:`Q_LAND_FINAL_ALT <Q_LAND_FINAL_ALT>` at :ref:`Q_WP_SPEED_DN <Q_WP_SPEED_DN>` until it reaches :ref:`Q_LAND_FINAL_ALT <Q_LAND_FINAL_ALT>`, at which point it continues to descend at :ref:`Q_LAND_SPEED <Q_LAND_SPEED>` until landing.
```
Comment (tridge) [inline plane/source/docs/qland-mode.rst]: should note that the position for the land is taken from the moment that QLAND is engaged

File: plane/source/docs/qloiter-mode.rst [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+-  Horizontal location can be adjusted with the the Roll and Pitch
+   control sticks with the default maximum horizontal speed being 5m/s
+   (see Tuning section below on how to adjust this).  When the pilot
+   releases the sticks the QuadPlane will slow to a stop.
+-  Altitude can be controlled with the Throttle control stick just as in
+   :ref:`QHOVER<qhover-mode>` mode.
+-  The heading can be set with the Yaw control stick
+
+The vehicle can be armed in QLOITER mode but only once the GPS has 3D lock and the HDOP has dropped below 2.0.  :ref:`More details on LED patterns here <common-leds-pixhawk>`.
```
Comment (tridge) [inline plane/source/docs/qloiter-mode.rst]: in plane the arming checks are the same in all modes

File: plane/source/docs/qloiter-mode.rst [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+
+Tuning
+======
+
+QLOITER mode incorporates the altitude controller from QHOVER mode.
+Details for tuning are in :ref:`QHOVER <qhover-mode>` mode description.
+
+For QuadPlane-3.6 (and higher)
```
Comment (tridge) [inline plane/source/docs/qloiter-mode.rst]: 3.6?

File: plane/source/docs/qrtl-mode.rst [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+    :target: ../_images/QRTL.jpg
+
+Alternatively, you may :ref:`configure the plane to return to a Rally Point <common-rally-points>`, rather than the home location.
+
+.. warning::
+
+   "Home" position is always supposed to be your Plane's actual
+   GPS takeoff location:
+
+   #. It is very important to acquire GPS lock before arming in order for
```
Comment (tridge) [inline plane/source/docs/qrtl-mode.rst]: plane requires this by default

File: plane/source/docs/qstabilize-mode.rst [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+   the vehicle in place as it is pushed around by the wind.
+-  Pilot's yaw input controls the rate of change of the heading.  When
+   the pilot releases the yaw stick the vehicle will maintain it's
+   current heading.
+-  Pilot's throttle input controls the average motor speed meaning that
+   constant adjustment of the throttle is required to maintain
+   altitude.  If the pilot puts the throttle completely down the motors
+   will go to their minimum rate (Q_M_SPIN_ARMED) and if the vehicle
+   is flying it will lose attitude control and tumble.
+-  The throttle sent to the motors is automatically adjusted based on
```
Comment (tridge) [inline plane/source/docs/qstabilize-mode.rst]: note that flying backwards in a quadplane is a very bad idea, due to reversal of fixed wing control surfaces leading to instability

**Comment by Hwurzburg** [comment]
Thanks Andrew.....changes made...merging

**Pull request merged**